### DISCUSSION TITLE
Where's the Infinity for Reddit apk?

### DISCUSSION THREAD

**Discussion by Sneffel** [Android-App-Patches/build-apps#78] [Q&A] [answered]
This may sound like a stupid question to a developer who's invested some time to understand the various branches of the ReVanced project. But I have not clue where to find official Infinity ReVanced apk. 
I can't find it inside the [usual URL here](https://github.com/revanced-apks/build-apps/releases). My suggestion would be to make everything more user friendly until the ReVanced patcher app starts working.

**Comment by rabilrbl** [comment]
@Sneffel If you found your answer, please close1

**Comment by Sneffel** [comment]
I haven't

**Reply by rabilrbl** [reply]
Please elaborate I didn't get your question

**Comment by rabilrbl** [comment, marked as answer]
Now new releases include infinity for reddit
